develop/tutorials/articles/230-web-experience-management/02-displaying-custom-assets-with-display-pages/01-creating-a-display-container.markdown

```diff
@@ -1,20 +1,42 @@
-# Creating a Display Container
+# Creating a Display Contributor
 
 All the logic related to Asset Display Contributors, such as reading fields and 
-their values, is contained in the `BaseAssetDisplayContributor` class. This 
+their values, is contained in the `AssetDisplayContributor` class. This 
 makes integrating it with your application as simple as creating a class that 
-extends `BaseAssetDisplayContributor` and implements it's `getClassName()` and `getClassTypeValue()` methods in your module.
+references the `AssetDislayContributor` service, extends 
+`BaseAssetDisplayContributor` and implements it's `getClassName()` and 
+`getClassTypeValue()` methods in your module.
 
 ## BaseAssetDisplayContributor
 
-Your custom `*AssetDisplayContributor` should implement 
-`BaseAssetDisplayContributor` and must provide the class name and and the 
+Your custom `*AssetDisplayContributor` must implement 
```

Comment:
Hi @stephenkostas, here we have two options as Pavel mentioned. If we implemente BaseAssetDisplayContributor, all the logic is available to you. But if you decide not to, and just implement the interface, you would be in charge of implementing all the logic